views/default/input/access.php

```diff
@@ -48,6 +48,17 @@
 }
 
 if (is_array($vars['options_values']) && sizeof($vars['options_values']) > 0) {
+
+	if ($vars['value'] && !array_key_exists($vars['value'], $vars['options_values']) && $container && $container->canEdit()) {
```

Comment:
I wonder if this code will cause a nasty merge conflict with #7766 (now in 1.x).